.github/workflows/build-and-publish.yml

```diff
@@ -25,13 +25,12 @@ jobs:
         working-directory: ${{ inputs.working-directory }}
     steps:
       - name: SCM Checkout
-        uses: actions/checkout@v4
+        uses: actions/checkout@v5
```

Comment:
The workflow changes comes from running `poetry run -- tbx workflow update` and checking that the custom modifications for the pytest-plugins are preserved. The modifications primarily come from there being multiple packages in the repositories, which necessitates that a working-directory is added for the steps..